### ISSUE TITLE
Auto check and create job table

### ISSUE THREAD

**Issue by kritzware** [Opteo/oxen-queue#5]
Instead of making the user call the method `createTable()`, the library could check if the table exists once connected and then call this function internally.

```javascript
await ox.createTable()
```

**Comment by avermeil** [comment]
That's interesting... is there much of a cost to calling 

```sql
CREATE TABLE IF NOT EXISTS oxen_queue
```
for every instance of Oxen?

**Comment by gaurav8298** [comment]
No, I don't see anything wrong with `create table if not exist`, rather this is a safe and good programming.
